src/modm/driver/adc/ads868x.lb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, Niklas Hauser
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+
+def init(module):
+    module.name = ":driver:ads868x"
+    module.description = """
+# ADS868x Driver.
+
+ADS868x 16-Bit, High-Speed, Single-Supply, SAR ADC Data Acquisition System
+With Programmable, Bipolar Input Ranges
```

Comment:
```suggestion
def init(module):
    module.name = ":driver:ads8681"
    module.description = """
# ADS8681/ADS8685/ADS8689 Driver.

ADS8681/ADS8685/ADS8689 16-Bit, Single-Channel, High-Speed, Single-Supply, SAR ADC Data Acquisition System
With Programmable, Bipolar Input Ranges

This driver does not work with ADS8684 and ADS8685 quad/octa channel ADCs.
```

Reply:
@salkinium (How) is it possible with [lbuild](https://github.com/modm-io/lbuild) to name this driver `:ads8681` and have aliases `:ads8685`/`:ads8689`?

Reply:
Yes, [lbuild has aliases](https://github.com/modm-io/lbuild#aliases). Though in this case we probably want a silent alias, not sure if that is supported currently, or if lbuild just prints an empty warning for it still.